torch/testing/_internal/distributed/rpc/dist_autograd_test.py

```diff
@@ -2747,7 +2847,11 @@ def test_gradients_synchronizations(self):
 
         if self.rank == 0:
             # this is master
-            layers = [nn.Linear(2000, 2000) for _ in range(self.world_size - 1)]
+            if sparse:
+                linear_size = 3
```

Comment:
shall we keep the linear size and change the input sparse tensor size?

Reply:
I changed the input to the function for the sparse test. 

```
    @skip_if_lt_x_gpu(4)
    def test_gradients_synchronizations_sparse(self):
        self._gradients_synchronizations(
            torch.randn(5000, 2000).to_sparse()
        )
 ```